src/tests/work-item/work-item-list/work-item-detail.page.js

```diff
@@ -311,6 +311,7 @@ value: function ()
 
   /* The following UI elements support the assignment of a user to a work item */
 
+<<<<<<< HEAD
```

Comment:
That line is left over from the merging and needs to be deleted. The build is failing due to it and I can't merge it.